### ISSUE TITLE
Changes for STR filters, new Stranger option and new loci for ExHu >4

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/MIP#1804]
Heads up for new STR features!

The [filters feature in Scout is essentially finished](https://github.com/Clinical-Genomics/scout/pull/2208) and will enable filtering of files annotated with Stranger v0.6.1 and later. If RankScores are not handled in other ways, it will bark on those though so I suggest jumping directly to [Stranger v0.7](https://github.com/moonso/stranger/releases/tag/v0.7). Cases annotated with older versions will still display, but will not filter due to the lack of a proper HGNC-id mapping. Note that there are also plenty of new clinically relevant loci in the latest variant-catalog distributed with Stranger. If you wish I could mirror them to references-files as well for convenience?

**Comment by jemten** [comment]
Hi there!
We're about to make a small update to MIP and could possibly squeeze this in. Currently MIP is running stranger v0.5.5 and expansionhunter v3.2.2. All that is required is an update to stranger or do you need expansionhunter to be updated as well? 
v0.7 introduced the option family_id which I can't find documentation on. Do you need MIP to start using that option?

**Comment by dnil** [comment]
Perfect! While it is not exactly required, I strongly suggest updating to exhu4 (.0.2 I guess) as well - especially the inexact repeats like polyAla disorders are much better resolved. Yep, thanks, I'll add a line to the docs! It is on the cli, so already in a way self documented. The option will be familiar from the other genmod-ish tools:
```
Usage: stranger [OPTIONS] VCF

  Annotate str variants with str status

Options:
  -f, --repeats-file PATH         Path to a file with repeat definitions. See
                                  README for explanation  [default: /Users/dan
                                  nil/sandbox/stranger/stranger/resources/vari
                                  ant_catalog_grch37.json]
  -i, --family_id TEXT
  --version
  --loglevel [DEBUG|INFO|WARNING|ERROR|CRITICAL]
                                  Set the level of log output.  [default:
                                  INFO]
  --help                          Show this message and exit.
```
How about the references; is it easier for you if I PR them to the reference-files repo as well, or is it all the same if you use them from the stranger repo directly?

**Comment by jemten** [comment]
Alright, I'll update expansionhunter as well and supply the case_id to stranger. Would be great if you could add the variant catalogs to the reference repo. This is the one we use today. https://raw.githubusercontent.com/Clinical-Genomics/reference-files/master/rare-disease/disease_loci/ExpansionHunter-v3.1.2/variant_catalog_grch37.json

**Comment by dnil** [comment]
Oki, done: https://github.com/Clinical-Genomics/reference-files/pull/38
Looking forward to it! 😸

**Comment by jemten** [comment]
Fixed in MIP v9.1.3
